mfhdf/test/tattdatainfo.c

```diff
@@ -480,25 +474,25 @@ add_sdsSDG_annotations()
 
     gen2Dfloat(ROWS, COLS, data);
 
-    ret = DFSDsetdims(2, dimsizes);
+    DFSDsetdims(2, dimsizes);
 
     /********  Write labels and descriptions *********/
     for (j = 0; j < REPS; j++) {
 
         /* write out scientific data set */
-        ret = DFSDadddata(DFAN_SDG_FILE, 2, dimsizes, (VOIDP)data);
+        DFSDadddata(DFAN_SDG_FILE, 2, dimsizes, (VOIDP)data);
 
         { /* write out annotations for 2 out of every 3 */
             refnum = DFSDlastref();
-            ret    = DFANputlabel(DFAN_SDG_FILE, DFTAG_SDG, refnum, labsds);
-            ret    = DFANputdesc(DFAN_SDG_FILE, DFTAG_SDG, refnum, descsds, (int32)HDstrlen(descsds));
+            DFANputlabel(DFAN_SDG_FILE, DFTAG_SDG, refnum, labsds);
+            DFANputdesc(DFAN_SDG_FILE, DFTAG_SDG, refnum, descsds, (int32)HDstrlen(descsds));
```

Comment:
Shouldn't we be checking the return values of these functions?

Reply:
Maybe, but someone would need to understand the calls (@bmribler ?) I just removed the unused variable. I can add a check though.

Reply:
Actually the bigger problem is this function, add_sdsSDG_annotations, vs the previous function, add_sdsNDG_annotations. The previous has a var num_errs, which gets returned. This function uses the global var, num_errs, and returns 0.